docs/src/modules/components/DemoToolbar.js

```diff
@@ -363,6 +364,71 @@ export default function DemoToolbar(props) {
     isFocusableControl,
   });
 
+  const devMenuItems = [];
+  if (process.env.STAGING === true) {
```

Comment:
I'm curious. Why not doing
```suggestion
  if (process.env.PULL_REQUEST === true) {
```

As we already do in https://github.com/mui-org/material-ui/blob/33850e52ca32787fbc02b64d3225bac2d451612c/docs/src/modules/utils/getDemoConfig.js#L7

Reply:
> I'm curious. Why not doing

Because I would use these links on all `material-ui.netlify.app` deploys

Reply:
Could you open a new issue explaining why it would make sense there? I don't understand why this is an issue besides "let's DRY out code at all cost".

Reply:
Yeah, please open a new issue. I don't understand how a link using a codesandbox deploy would look outside of PRs. If it's not about DRY then I understand even less why this is raised here.

Reply:
I don't see codesandbox-ci build for the baseline branch, It might not work 😁. For the issue, I don't have the incentive to open one (I didn't see a strong pain it could solve). I think that we can simply snooze it.